docs/source/beaker.rst

```diff
@@ -106,6 +106,8 @@ describes the available recipesets options.
 +---------------------+------------+----------+-----------------------------------------+
 | ssh_key             | false      | list     | list of strings                         |
 +---------------------+------------+----------+-----------------------------------------+
+| kickstart           | false      | string   | N/A                                     |
```

Comment:
Should we specify that this should be an absolute path to a kickstart template?

Reply:
I may definitively be useful. Does the last column fit this purpose?

Reply:
yes - that would be fine.  Thank you for the patch

Reply:
updated with description